crates/editor/src/element.rs

```diff
@@ -5215,15 +5290,18 @@ mod tests {
         let expected_invisibles = vec![
             Invisible::Tab {
                 line_start_offset: 0,
+                line_end_offset: 4,
```

Comment:
Can we use the `TAB_SIZE` constant defined above instead of specifying `4` as a literal?

Reply:
Done! I've updated it to match in the other tests as well.